config/locales/zh.yml

```diff
@@ -1447,8 +1447,8 @@ notices.piv_cac_configured: 添加您的政府雇员ID
 notices.privacy.privacy_act_statement: 隐私法声明
 notices.privacy.security_and_privacy_practices: 安全实践和隐私法声明
 notices.resend_confirmation_email.success: 我们发送了另外一个确认电邮。
-notices.session_cleared: 为了你的安全，如果你在 %{minutes} 分钟内不移动到一个新页面，我们会清除你输入的内容。
-notices.session_timedout: 我们已将你登出。为了你的安全，如果你在 %{minutes} 分钟内不移动到一个新页面，%{app_name} 会结束你此次访问。
+notices.session_cleared: 出于安全考虑，如果你在 %{minutes} 分钟内不移动到一个新页面，我们会清除你输入的内容。
```

Comment:
Beginning of the flash message:
![Screenshot 2024-11-06 at 5 25 06 PM](https://github.com/user-attachments/assets/9213f826-ca43-47c1-87e7-1aaf87b49661)

N.B.: This comes up when the user isn't logged in (such as during sign-up), because it doesn't make sense to "log out" someone who hasn't finished signing up. The string below is for logged-in users.